src/Pods/AppServer/Startup.cs

```diff
@@ -47,5 +52,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 routes.MapHub<BenchHub>(HUB_NAME);
             });
         }
+
+        private void AddComandHandlers(string connectionString)
+        {
+            const string sender = "AppServer";
+            var crash = MessageHandler.CreateCommandHandler(Commands.General.Crash, cmd =>
+               {
+                   Environment.Exit(1);
+                   return Task.CompletedTask;
+               });
+            Task.Run(async () => await MessageClient.ConnectAsync(connectionString, sender, crash));
```

Comment:
no reference for MessageClient, gc will collect it...

Reply:
fixed